workflow-templates/zephyr/README.md

```diff
@@ -85,6 +87,13 @@ They can be used in two different scenarios:
 - During a release process
 - In Pull Requests
 
+## Self hosted runners
+
+The `build.yaml` and `twister.yaml` workflows are configured to run on self-hosted GitHub runners. These runners are hosted on the internal Irnas server. 
+
+To change the runner configuration back to the default GitHub runners, modify the `runs-on` field in the workflow file. 
+This can be done by running the `sed -i 's/self-hosted/ubuntu-20.04/' /workflow-templates/zephyr/.github/workflows/*.yaml` command from the root directory of this repository.
+
```

Comment:
This section is not on the appropriate place, since it cuts _How to Use_, _Release process_, _Pull requestes_ sections apart.

Please place it somewhere else, I would suggest at the end of the document (as it is not relevant for the normal use case).

Reply:
Additionally,  this line `sed -i 's/self-hosted/ubuntu-20.04/' /workflow-templates/zephyr/.github/workflows/*.yaml won't work for the user, as it references the `workflow` repository which should stay the same.

Change the instruction so that it is appropriate for the user.

Reply:
Responded in IRL, the command should be:
```
sed -i s/self-hosted/ubuntu-20.04/ .github/workflows/*.yaml
```